### ISSUE TITLE
spam limiter triggering for no (apparent) reason

### ISSUE THREAD

**Issue by Nahor** [SteelSeries/gamesense-sdk#124]
When launching my app within 5 min (320s actually) of launching SteelSeriesGG, when trying to bind an event, I get an error `Events for too many games have been registered recently, please try again later`

Here is the logs from a modified version of [my app](https://github.com/Nahor/sseClock):

```
2022-05-16 17:00:13 - Using address: http://127.0.0.1:64691
2022-05-16 17:00:13 - *** Sending request: http://127.0.0.1:64691/game_metadata
2022-05-16 17:00:14 - *** Sending request: http://127.0.0.1:64691/bind_game_event
2022-05-16 17:00:14 - Url: http://127.0.0.1:64691/bind_game_event
2022-05-16 17:00:14 -    Status code: 400 - HTTP/1.1 400 Bad Request
2022-05-16 17:00:14 -    Content-type: text/html; charset=utf-8
2022-05-16 17:00:14 -    Body: {"error": "Events for too many games have been registered recently, please try again later"} 
```

**Comment by JHunz** [comment]
We're actually already aware of the reason for this, and a fix will be released in an upcoming version.  I'll try to remember to circle back and update the issue when that release comes out.

**Comment by Nahor** [comment]
Seems to be fixed in 20.0.0.

**Comment by JHunz** [comment]
Yes, it should be.  Thanks for reminding me
